tm2/pkg/bft/config/toml.go

```diff
@@ -49,256 +38,20 @@ func LoadConfigFile(path string) (*Config, error) {
 /****** these are for production settings ***********/
 
 // WriteConfigFile renders config using the template and writes it to configFilePath.
-func WriteConfigFile(configFilePath string, config *Config) {
-	var buffer bytes.Buffer
+func WriteConfigFile(configFilePath string, config *Config) error {
+	// Marshal the config
+	configRaw, err := toml.Marshal(config)
```

Comment:
Is there any advantage to using `comment` struct annotations here so that the toml marshaling produces a file with comments similar to how it existed before?

Reply:
You and @thehowl are absolutely right, there is value in preserving field explanations 💯 

Added:

[84b627b](https://github.com/gnolang/gno/pull/1544/commits/84b627b11278ce42e49700b67d566a9dfbe1db86)